src/Containers/CadView.jsx

```diff
@@ -67,8 +66,12 @@ export default function CadView({
   const isItemPanelOpenState = {value: isItemPanelOpen, set: setIsItemPanelOpen}
   const [isLoading, setIsLoading] = useState(false)
   const [model, setModel] = useState(null)
-  const snackMessage = useStore((state) => state.snackMessage)
+  const setViewerStore = useStore((state) => state.setViewerStore)
+  const setModelStore = useStore((state) => state.setModelStore)
```

Comment:
Seems like we'll have a bunch of variable like this that have react + zustand refs.  The current name reads a little backwards to me.. does setX and storeX sound good to you?